### PR TITLE
Edited the about description

### PR DESCRIPTION
**Pull request by AwuorMarvin**

### CODE REVIEW

**Comment by brayoh** [comment]
the code looks okay but try improve on your indentation, code is read more than its written so your code should be well indented and readable, they are some plugins for various editors to automatically indent your code you can check them out too

File: templates/about.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <h1>About US</h1>
         <p class="lead">The Bucket is all about giving you the opportunity to maximise your potential and 
             get to experience all which you have always dreamt of. It's a proprietary service that elps you keep track of all 
-            the exhilirating activities that not many get to experience. 
+            the exhilirating activities that not many get to experience. Sign up today and give it a spin.
         </p>
```
Comment (brayoh) [inline templates/about.html]: indent this block

**Comment by JayKay24** [comment]
Nice work bro. Perhaps you could remove that redundant piece of text about an exhilarating experience

File: templates/about.html (hunk 1 above)
Comment (AwuorMarvin) [reply templates/about.html]: Okay, let me do that right away.

**Comment by brayoh** [comment]
@JayKay24  the code is removed, red line means removed, green is changes made

**Review by HawiCaesar — COMMENTED** [review]
I think Its good

File: templates/about.html [hunk 2]
```diff
@@ -3,10 +3,9 @@
 {% block body %}
     <div class="jumbotron text-center">
         <h1>About US</h1>
-        <p class="lead">The Bucket is all about giving you the opportunity to maximise your potential and 
-            get to experience all which you have always dreamt of. It's a proprietary service that elps you keep track of all 
-            the exhilirating activities that not many get to experience. Sign up today and give it a spin.
-        </p>
+            <p class="lead">The Bucket is all about giving you the opportunity to maximise your potential and 
+                get to experience all which you have always dreamt of. Sign up today and give it a spin.
+            </p>
```
Comment (brayoh) [inline templates/about.html]: it looks okay now, preferably use 4 spaces or 2 tabs depending on your settings to indent html code here as an example:
`<p class="lead">
     The Bucket is all about giving you the opportunity to maximise your potential and get 
      to experience all which you have always dreamt of. Sign up today and give it a spin.
   </p>`

Comment (AwuorMarvin) [reply templates/about.html]: Okay. I'll try and do all that when refactoring. For now I was just trying to get the hang of creating and merging a pull request. :)

**Comment by JayKay24** [comment]
@brayoh thanks for the info

**Comment by brayoh** [comment]
@JayKay24 its cool bro.

**Pull request merged**